nomad/structs/structs.go

```diff
@@ -5817,6 +5817,9 @@ var (
 	}
 )
 
+// TODO Figure out if we can remove this. This is our fork that is just way
```

Comment:
Should we open a separate issue/task to investigate this?

Reply:
Yeah I was just going to do it as a follow up PR but I can track this